lib/ui/myplot/my_plot_page.dart

```diff
@@ -2,10 +2,99 @@ import 'package:farmsmart_flutter/data/model/crop_entity.dart';
 import 'package:farmsmart_flutter/model/loading_status.dart';
 import 'package:farmsmart_flutter/redux/app/app_state.dart';
 import 'package:farmsmart_flutter/redux/home/myPlot/my_plot_actions.dart';
-import 'package:farmsmart_flutter/ui/myplot/my_plot_child_item.dart';
 import 'package:farmsmart_flutter/ui/myplot/myplot_viewmodel.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_redux/flutter_redux.dart';
+import 'new_my_plot_child_item.dart';
+
+abstract class HomeMyPlotPageStyle {
```

Comment:
Let´s think of naming in terms of the model and not the design. E.g the current design has this plot list as the home tab, however that may change. let's think of each widget independently. So I would name this "PlotList" (as its simply a list of plots, I may want to have a list of plots elsewhere in the app, so this should be built and named in a way which can be reused with maybe a slight style change...